core/ajax/jMQTT.ajax.php

```diff
@@ -65,8 +65,22 @@
 
     ajax::init();
 
+    if (init('action') == 'getOldTemplateList') {
```

Comment:
Why do we keep get**Old**TemplateList?

Reply:
Removed in commit https://github.com/Domochip/jMQTT/pull/112/commits/2d65645dcec8b1e5464e4f5c01e4f8328054189c